rasa/shared/utils/io.py

```diff
@@ -366,22 +366,23 @@ def dump_obj_as_json_to_file(filename: Union[Text, Path], obj: Any) -> None:
     write_text_file(json.dumps(obj, indent=2), filename)
 
 
-def _dump_yaml(obj: Dict, output: Union[Text, Path, StringIO]) -> None:
-    import ruamel.yaml
+def dump_obj_as_yaml_to_string(
+    obj: Dict, should_preserve_key_order: bool = False
+) -> Text:
+    """Writes data (python dict) to a yaml string.
 
-    yaml_writer = ruamel.yaml.YAML(pure=True, typ="safe")
-    yaml_writer.unicode_supplementary = True
-    yaml_writer.default_flow_style = False
-    yaml_writer.version = YAML_VERSION
+    Args:
+        obj: The object to dump. Has to be serializable.
```

Comment:
It's already a dict according to the type hint, do you think we should mention `serializable`?

Reply:
type hint was wrong 👍 Can also be `list` or `Text` or whatever